pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -772,6 +772,14 @@ func validateSoundDevices(field *k8sfield.Path, spec *v1.VirtualMachineInstanceS
 				Field:   field.Child("Sound").String(),
 			})
 		}
+		name := spec.Domain.Devices.Sound.Name
+		if name == "" {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: fmt.Sprintf("Sound device requires a name field."),
+				Field:   field.Child("Sound").String(),
```

Comment:
I think the field should be `field.Child("Domain").Child("Devices").Child("Sound")` or better pass the correct field to function.  It would easy the search for the right field for user.

But I think this is general problem in this file.